de_docs/getting-started/user-guides/dcrwallet-tickets.md

```diff
@@ -0,0 +1,208 @@
+# **Buying Tickets With dcrwallet**
+
+Last updated for v0.8.2 on 4/18/2017
+
+This guide is intended to walk through ticket buying using `dcrwallet`. It will cover both manual ticket purchase and automatic ticket purchase for solo-voting and stakepool-voting configurations.
+
+**Prerequisites:**
+
+- Use either the latest [dcrinstall](/getting-started/install-guide.md#dcrinstall) to install `dcrctl`. Additional steps will be required if another installation method was used.
```

Comment:
dcrinstall install all 3 cmd line tools, not just dcrctl.  Do you want the user to get all, or just dcrctl?